src/signals/incident/components/IncidentPreview/components/DateTime/index.js

```diff
@@ -9,12 +12,12 @@ const getValue = (value, incident) => {
     return '';
   }
 
-  const time = moment(`${incident.incident_time_hours}:${incident.incident_time_minutes}`, 'H:m').format('H:mm');
+  const time = `${incident.incident_time_hours}:${String(incident.incident_time_minutes).padStart(2, '0')}`;
   if (incident.incident_date === 'Vandaag') {
     return `Vandaag, ${time}`;
   }
 
-  return `${moment(incident.incident_date).format('dddd D MMMM')}, ${time}`;
+  return `${capitalizeFirstLetter(format(parseISO(incident.incident_date), 'EEEE d MMMM', { locale: nl }))}, ${time}`;
```

Comment:
The `capitalizeFirstLetter` function can be handled by CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform#:~:text=The%20text%2Dtransform%20CSS%20property,help%20improve%20legibility%20for%20ruby.

Reply:
Indeed! I have tried that but in this case the different value types are renderd from an array and it would make the change more complex than this.